src/js/common/helpers/environment.js

```diff
@@ -0,0 +1,13 @@
+const _Environments = {
+  staging: { API_URL: 'https://dev.vets.gov/api', BASE_URL: 'https://dev.vets.gov/' },
+  production: { API_URL: 'https://dev.vets.gov/api', BASE_URL: 'https://dev.vets.gov/' },
+  development: { API_URL: 'http://localhost:4000', BASE_URL: 'http://localhost:3000' },
```

Comment:
running locally `vets-api` defaults to port `3000` and `vets-website` to `3001` - let's change the development routes to match that and then the default config will just work when developing locally?